src/python/test/test_point_cloud.py

```diff
@@ -0,0 +1,35 @@
+from gudhi.point_cloud.timedelay import TimeDelayEmbedding
+
+def test_normal():
```

Comment:
The test looks good. Maybe we could name the file test_time_delay.py so it is more specific?

Reply:
I want to change the file name from test_point_cloud.py to test_time_delay.py.

Reply:
Sure, you can go ahead, it is just `git mv test_point_cloud.py test_time_delay.py` (then commit and push).

Reply:
I changed a file name.

Reply:
You also need to modify CMakeLists.txt so it tests the new name instead of the old name.

Reply:
I succeed test in new name(test_time_delay.py).  
- Test result
![new_name_test_2020-02-07](https://user-images.githubusercontent.com/58589594/74025597-aa338e80-49e7-11ea-94b6-9198ad4f04d0.png)